.github/workflows/deploy-ncp.yml

```
@@ -0,0 +1,108 @@
+          export DB_HOST=${{ secrets.DB_HOST }}
+          export DB_PORT=${{ secrets.DB_PORT }}
+          export DB_NAME=${{ secrets.DB_NAME }}
+          export DB_USER=${{ secrets.DB_USER }}
+          export DB_PASSWORD=${{ secrets.DB_PASSWORD }}
+          export REDIS_HOST=${{ secrets.REDIS_HOST }}
+          export REDIS_PORT=${{ secrets.REDIS_PORT }}
+          export REDIS_PASSWORD=${{ secrets.REDIS_PASSWORD }}
+          export JWT_SECRET_KEY=${{ secrets.JWT_SECRET_KEY }}
+          export NAVER_CLIENT_ID=${{ secrets.NAVER_CLIENT_ID }}
+          export NAVER_CLIENT_SECRET=${{ secrets.NAVER_CLIENT_SECRET }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)